SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -810,8 +810,8 @@ Plotter {
 }
 
 + Wavetable {
-	plot { |name, bounds, minval, maxval, separately = false|
-		^this.asSignal.plot(name, bounds, minval: minval, maxval: maxval, separately: separately)
```

Comment:
sorry, should have explained in the comment : 

the arg 'separately' applies to multichannel arrays - it sets  whether to determine plot ranges separately per sub-array or not. Signal and Wavetable are subclasses of FloatArrays and thus per definition single arrays always. thus setting 'separately' has no effect, and is a potential source of confusion. thus, it seems clearest and least error-prone to remove this argument. 
generally, plot is nearly the same for very many collections, so it seems best to point to the general doc in the plot help file rather than duplicating all the doc details in several places.